<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>628</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>628</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>890</width>
    <height>628</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resource/icon.png</normaloff>../resource/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>821</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
     </font>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="preview">
     <attribute name="title">
      <string>          preview          </string>
     </attribute>
     <widget class="QWebEngineView" name="webView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>791</width>
        <height>311</height>
       </rect>
      </property>
      <property name="url">
       <url>
        <string>about:blank</string>
       </url>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Preview Process</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="preview_process">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>671</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="download">
     <attribute name="title">
      <string>download</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Download Info</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Save to&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Stream&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="path">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>20</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_start">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>230</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Go!</string>
       </property>
      </widget>
      <widget class="QComboBox" name="stream_comboBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>291</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_stop">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>230</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QToolButton" name="btn_path">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>80</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>16</width>
        <height>271</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>361</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <widget class="QTextBrowser" name="log">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>361</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Process</string>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>20</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="download_process">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>651</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="basic_info">
     <attribute name="title">
      <string>          Service          </string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>761</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>50</y>
         <width>381</width>
         <height>101</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../resource/logo_youtudu.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>301</width>
         <height>291</height>
        </rect>
       </property>
       <property name="title">
        <string>Basic Info</string>
       </property>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>281</width>
          <height>191</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="LogIN">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>230</y>
          <width>301</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>.Apple SD Gothic NeoI</family>
         </font>
        </property>
        <property name="text">
         <string>Youtube Log In</string>
        </property>
       </widget>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>0</y>
         <width>20</width>
         <height>311</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>180</y>
         <width>401</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Version : ver 1.0.1&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;This is from the RedoC Edu Project.&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;2020, Redoc Team, &lt;span style=&quot; font-weight:600;&quot;&gt;All right reserved&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="url">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;URL&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>160</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>351</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../resource/logo_youtudu.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
